src/utils.ts

```diff
@@ -43,14 +43,17 @@ const ENS_ABI = [
   'function text(bytes32 node, string calldata key) external view returns (string memory)',
   'function resolver(bytes32 node) view returns (address)' // ENS registry ABI
 ];
-const SONIC_ABI = [
-  'function ownerOf(uint256 tokenId) external view returns (address address)'
+const UD_MAPPING = {
```

Comment:
UD is confusing; maybe we should just use `Unstoppable` directly everywhere

Reply:
Unstoppable is incomplete, I think UD is fine